src/main/java/core/basesyntax/service/impl/FileWriterCsvImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterCsvImpl implements FileWriter {
+    private static final String REPORT_FILE_NAME = "src/main/resources/report.csv";
+
+    @Override
+    public void writeToFile() {
+        try (BufferedWriter bufferedWriter
+                     = new BufferedWriter(new java.io.FileWriter(REPORT_FILE_NAME, true))) {
+            for (Map.Entry<String, Integer> entry: Storage.STORAGE.entrySet()) {
+                bufferedWriter.write(entry.getKey() + "," + entry.getValue());
+                bufferedWriter.newLine();
+            }
```

Comment:
Only writing should be here
![image](https://user-images.githubusercontent.com/20326858/236444207-be29c57d-4e5f-43ab-8062-4daa6da02f1a.png)